### ISSUE TITLE
Low inference time on my own dataset

### ISSUE THREAD

**Issue by neneyork** [fudan-zvg/4d-gaussian-splatting#38]
Thanks for your work on dynamic gaussian splatting. I am testing the 4DGS on my own dataset, which is an single view dataset with resolution 512*640. The result are advanced in image quality, however, the inference time is 23fps. While the inference time on 4DGS based on the hexplane is 61fps, which is not consistent to the result in your articel. Is this due to the dataset?

**Comment by EarltShirt** [comment]
Hey,
sorry to bother you, quick question. When processing your own dataset, were you able to get good results (for example a good PSNR, a decrease in loss, etc) on the test image set ? During training, my train data's PSNR is quite high ( around 45) and the visual results are okay, but on the test set the PSNR is never able to get over 11 ... What kind of scene are you using and could you send me the hyperparameters you used.

Thanks a lor !

**Comment by neneyork** [comment]
> Hey, sorry to bother you, quick question. When processing your own dataset, were you able to get good results (for example a good PSNR, a decrease in loss, etc) on the test image set ? During training, my train data's PSNR is quite high ( around 45) and the visual results are okay, but on the test set the PSNR is never able to get over 11 ... What kind of scene are you using and could you send me the hyperparameters you used.
> 
> Thanks a lor !

Hi, the dataset that I use is a signle-view monocular video. I only changed the training iteration to 10_000 and reduce densification  times. It seems that your model is overfittinf, science the result is good on train dataset. Maybe you can train to reduce the training iteration. Hope this would help you.

**Comment by Alexander0Yang** [comment]
Hi @neneyork. 
Thanks for your interest in our work. It seems difficult to determine the cause of low inference speed based on the available information. May I ask whether you have compared the number of total Gaussian points after optimization?

**Comment by neneyork** [comment]
@Alexander0Yang Hi, I find the reason. The Hexplane based 4DGS render the dataset 50 times, the inference speed of first time is 6 fps, however, that of 50th time is 82 fps. The average is chosen asd the final inference speed. The result is consist with your paper if take the inference speed of first time, but I am not sure why the it would increase.
